discord/flags.py

```diff
@@ -1030,6 +1120,18 @@ class MemberCacheFlags(BaseFlags):
                Returns an iterator of ``(name, value)`` pairs. This allows it
                to be, for example, constructed as a dict or a list of pairs.
 
+        .. describe:: x + y
```

Comment:
Why is this here for the documentation of a specific flag

Reply:
It's under `MemberCacheFlags`

Reply:
Shouldn't _MemberCacheFlags(BaseFlags)_ get the operators / methods from base flags already?

Reply:
> Shouldn't _MemberCacheFlags(BaseFlags)_ get the operators / methods from base flags already?

It'll get them, they just won't be in the documentation unless they're put there

Reply:
Wouldn't autodoc be a good approach? Or doesn't it work.

Reply:
autodoc is for detecting modules/classes and their methods and automatically adding them to the documentation, it won't merge docstrings from two different classes

Reply:
What the.. that sounds so useless. 
Well ok fair.